enclave_build/src/lib.rs

```diff
@@ -12,10 +12,14 @@ mod yaml_generator;
 use docker::DockerUtil;
 use eif_defs::EIF_HDR_ARCH_ARM64;
 use eif_utils::{EifBuilder, EifIdentityInfo, SignEnclaveInfo};
+use serde_json::json;
 use sha2::Digest;
 use std::collections::BTreeMap;
+use tokio::runtime::Runtime;
 use yaml_generator::YamlGenerator;
 
+pub const DEFAULT_TAG: &str = "1.0";
```

Comment:
The default tag is usually `latest`, which doesn't make a lot of sense in this case. I suggest using the SHA256 value of the image?

Actually, better use the SHA256 value anyway, since tags are mutable.

Reply:
done